### DISCUSSION TITLE
How to focus on newly inserted extensions in editor?

### DISCUSSION THREAD

**Discussion by ImGelu** [ueberdosis/tiptap#2810] [General Questions]
Hi all,

I'm working on a Tiptap extension for Mathlive, a math editor. I managed to insert it into the Tiptap editor, but I can't seem to able to figure out how to fire a focus event. All I could find is how to fire the focus event on the *editor*

Can someone help me with this?

Thanks!

**Comment by sereneinserenade** [comment]
Please share a codesandbox, I can take a look into your problem
